src/types/ContractFunctionResult.ts

```diff
@@ -0,0 +1,23 @@
+import {ContractLogInfo, contractLogInfoListToSdk} from "./ContractLogInfo";
+import {ContractFunctionResult as ProtoContractFunctionResult} from "../generated/ContractCallLocal_pb";
+import {ContractId, contractIdToSdk} from "./ContractId";
+
+export type ContractFunctionResult = {
+    contractId: ContractId;
+    contractCallResult: Uint8Array | string;
+    errorMessage: string;
+    bloom: Uint8Array | string;
```

Comment:
Same for `contractCallResult` and here, just use `Uint8Array` and then `.get*_asU8()`.